tests/unit/info.tcl

```diff
@@ -6,9 +6,64 @@ proc errorstat {cmd} {
     return [errorrstat $cmd r]
 }
 
+proc latency_percentiles_usec {cmd} {
+    return [latencyrstat_percentiles $cmd r]
+}
+
+proc latency_hist_usec {cmd} {
+    return [latencyrstat_hist $cmd r]
+}
+
 start_server {tags {"info" "external:skip"}} {
     start_server {} {
 
+        test {latencystats: disable/enable} {
```

Comment:
Maybe we can test the stats for modules 2?

Reply:
@ranshid on the latest commit I've added the latency percentiles check to one of the modules test. This should do it: https://github.com/redis/redis/pull/9462/files#diff-682a213e9ff5ac20de6dd50890bf1cbeec8fe812acfd1a92d0e868c072472055R34